server/src/main/java/com/google/coffeehouse/servlets/JoinClubServlet.java

```diff
@@ -69,15 +70,15 @@
   private static final Gson gson = new Gson();
   private static final HttpTransport transport = new NetHttpTransport();
   private static final GsonFactory jsonFactory = GsonFactory.getDefaultInstance();
-  private final GoogleIdTokenVerifier verifier;
```

Comment:
Remove unused import for GoogleIdTokenVerifier on line 19.

Reply:
Thank you! Remove this import and others :).